src/components/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export function Header(): JSX.Element {
+  return (
+    <header className="p-3 mb-2" style={{ background: '#009cab' }}>
+      <HeaderLink to="/">
+        <HeaderLogo />
+      </HeaderLink>
+      <HeaderLink to="/">Tracks</HeaderLink>
+      <HeaderLink to="/stories">Stories</HeaderLink>
+    </header>
+  )
+}
+
+function HeaderLogo(): JSX.Element {
+  return (
+    <img
```

Comment:
Any reason not to inline this in the header, rather than making a new element? (I don't know react so it's a genuine question)

Reply:
I mostly did this due to the way all elements in the header are at the same somewhat higher level:

```jsx
<HeaderLink to="/">
    <HeaderLogo />
</HeaderLink>
<HeaderLink to="/">Tracks</HeaderLink>
<HeaderLink to="/stories">Stories</HeaderLink>
```

I'm no React expert either, so I don't know what's wise

Reply:
Elements are cheap. Use them. It's now much easier to switch for an svg than if you had inlined it.